dsl/config/branch.yaml

```
@@ -60,6 +60,9 @@ maven:
     staging_profile_id: 1976dd1c18392a
     build_promotion_profile_id: ea49ccd6f174
   artifacts_repository: ''
+  local_zipped_artifacts_repository:
+    url: https://repository.jboss.org/nexus/content/repositories/snapshots/
+    creds_id: kie-ci1-upload
```

I think you should use the already existing `artifacts_repository` and add a `artifacts_repository_credentials_id`
```suggestion
  artifacts_repository: https://repository.jboss.org/nexus/content/repositories/snapshots/
  artifacts_repository_credentials_id: kie-ci1-upload
```

---

forget about this